frontends/main/src/common/urls.ts

```diff
@@ -14,16 +16,16 @@ export const HOME = "/"
 
 export const ONBOARDING = "/onboarding"
 
-export const LEARNINGPATH_LISTING = "/learningpaths/"
-export const LEARNINGPATH_VIEW = "/learningpaths/:id"
+export const LEARNINGPATH_LISTING = "/learningpaths"
```

Comment:
NextJS by default canonicalizes removes trailing `/` from paths.

If we prefer to keep them, we can configure that with `next.config.js` (which affects the behavior of NextJS server and of the `Link` component). But it's slightly annoying since the jest tests aren't aware of next.config.js.